### DISCUSSION TITLE
I haven't got my benefits within 5 days in GitHub Student Developer Pack program

### DISCUSSION THREAD

**Discussion by danchistyakov** [community/community#141654] [GitHub Education]
### Hi everyone,

I haven't got my benefits within 5 days in GitHub Student Developer Pack program. What I'm doing wrong? I was last verified on Oct 9, 2024.

I see this:
approved Submitted Oct 9, 2024
Congrats! Your request for @danchistyakov was approved and your benefits will be available soon.

**Comment by davevad93** [comment]
Hi @danchistyakov ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @danchistyakov was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
